src/OrchardCore.Modules/OrchardCore.Resources/Assets.json

```diff
@@ -1,6 +1,5 @@
 [
   {
-    "generateRTL": true,
```

Comment:
Can we set it to `false` instead

Reply:
Yeah that makes sense, so done. Shame we can't comment in the `Assets.json` files. 

I just do the PR @hishamco because I couldn't understand if you knew what I was meaning or not, and it is just easier to do.

Have a look at the code for the markdown and html editors and you will see they don't use CodeMirror

Reply:
But we do use CodeMirror in the admin. And I checked the Liquid editor views and they use the manifests from OrchardCore.Resources. I think we should create an rtl-version instead? Or have a separate module that contains the assets for admin views.

Another opinion: You didn't remove it for trumbowyg, so why did you remove it for codemirror, are you actually using codemirror on the frontend for one of your projects and just thought about fixing this one?

Reply:
> But we do use CodeMirror in the admin

Yep, I remember I did some changed on it, let me revise my PRs and come back to you

Reply:
@deanmarcussen have a look to these #3575 #3669

Reply:
Yes, we do use CodeMirror in TheAdmin. But see above. We have a codemirror sass file in the admin that it builds from (and then builds rtl from that)

I only removed it for codemirror because it has a CDN listed. Totally happy to remove the CDN as an option.

But liquid module, for example, and others uses **only scripts** not styles. Styles all come from Admin
```
<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
<script asp-name="codemirror-mode-javascript" at="Foot"></script>
<script asp-name="codemirror-addon-display-autorefresh" at="Foot"></script>
<script asp-name="codemirror-addon-mode-simple" at="Foot"></script>
<script asp-name="codemirror-addon-mode-multiplex" at="Foot"></script>
<script asp-name="codemirror-mode-xml" at="Foot"></script>
<script asp-src="~/OrchardCore.Liquid/codemirror/liquid.js" at="Foot"></script>
```

Honestly the only reason I suggest fixing it is for the CDN values, because I was saying to Seth on https://github.com/OrchardCMS/OrchardCore/issues/4138 how TheAdmin styles are constructed, and something seem weird when I see that bootstrap resources is being built in rtl mode, plus has a cdn setting.

Reply:
Didn't realize these ones were just scripts that are not affected. GTG.
But I still think we should probably have another set of resources for the admin, to prevent any confusion, or a conflict in terms of versions... later.